benchmarks/base_classes.py

```diff
@@ -162,6 +162,25 @@ def run_inference(self, pipe, args):
             guidance_scale=1.0,
         )
 
+    def benchmark(self, args):
```

Comment:
This ensures that the overriden `get_result_filepath()` in the `LCMLoRATextToImageBenchmark` class gets called properly.